assets/js/blocks-registry/payment-methods/express-payment-method-config.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * Internal dependencies
+ */
+import {
+	assertConfigHasProperties,
+	assertValidPaymentMethodComponent,
+} from './assertions';
+
+export default class ExpressPaymentMethodConfig {
+	constructor( config ) {
+		// validate config
+		ExpressPaymentMethodConfig.assertValidConfig( config );
+		this.id = config.id;
+		this.activeContent = config.activeContent;
+		this.canMakePayment = config.canMakePayment;
+		Object.freeze( this );
```

Comment:
Hmm ya,  I did some reading and there does seem to be a quite a few articles/threads suggesting stay away from using `Object.freeze()` or `Object.seal()` in production code because of performance impact ([recent one](https://github.com/automerge/automerge/issues/177)).  Since this has marginal benefit I think I'll go ahead and remove this.  Thanks!